src/statespaceset.jl

```diff
@@ -262,3 +264,76 @@ function standardized_timeseries(d::AbstractStateSpaceSet)
     end
     return xs, means, stds
 end
+
+
+#####################################################################################
+#                          covariance/correlation matrix                            #
+#####################################################################################
+import Statistics: cov, cor
+using Statistics: mean, std
+using StaticArraysCore: MMatrix, MVector, SMatrix, SVector
+
+"""
+    cov(d::StateSpaceSet) → m::SMatrix
+
+Compute the covariance matrix `m` from the columns of `d`, where `m[i, j]` is the covariance
+between `d[:, i]` and `d[:, j]`.
+"""
+cov(x::AbstractStateSpaceSet) = fastcov(x.data)
```

Comment:
we should also replace all instances of `x.data` here with `vec(x)` that does the same thing but on the basis of functions.